book/website/project-design/persona/learnerneeds.md

```diff
@@ -0,0 +1,226 @@
+Key ideas covered in this chapter:
+- [Designing your learning products](#designing-your-learning-products)
+    - [Start with research](###Start-with-research) 
+    - [Show and Tell](###Show-and-Tell)
+    - [Empathy](###Empathy)
+    - [Never assume, deconstruct bias](Never-assume,-deconstruct-bias)
+    - [Include everyone, accessibility (eg. neurodiversity)](###Include-everyone,-accessibility-(eg.-neurodiversity))
+- [Learning Persona](##Learning-Persona)
+    - [Group Activities to Identify Learner Needs](###Group-Activities-to-Identify-Learner-Needs)
+        - [Learner Surveys](####Learner-Surveys)
+        - [Feedback Circles](####Feedback-Circles)
+        - [Brainstorming sessions](####Brainstorming-sessions)
+- [ Competency Profiles and Professional Standards](##Competency-Profiles-and-Professional-Standards)
+    - [What is a competency?](###What-is-a-competency?)
+    - [Key aspects of the competency-based approach](###Key-aspects-of-the-competency-based-approach)
+    - [Competency frameworks and accreditation](###Competency-Frameworks-and-Accreditation)
+        - [T-Shaped competency framework](####T-Shaped-Competency-Framework)
+        - [Professionalization and Accreditation of Data Science](####Professionalization-and-Accreditation-of-Data-Science)
+        - [AdvanceHE: Professional Standards Framework for Educators -  A Case Study ](####AdvanceHE:-Professional-Standards-Framework-for-Educators--A-Case-Study)
+
+## Designing your learning products
+Among the possible ways of starting your teaching material design, a great consensus in the educators cohorts is to focus on your user experience, using the so called UX principles. In the following we  identify and describe 5 UX principles that can be pitched for your special kind of users: the 'learners'.
+### Start with research
+To design effective educational content, begin by conducting thorough research. early your overall preparation. First step could be to form a clear picture of the topics your delivery will cover, making sure to touch on all the relevant problems and questions related to it (Key Topics Identification). Which are the relevant questions you should address? A way to find out is to research such questions and also brainstorm with your team (Relevant Questions). If you do not have one, you could share a document with colleagues or expert of the field for feedback and discussion (Feedback and Discussion). It is also very important to clarify from the beginning how you would measure the success of your product (Success Metics). After this phase, you could start specifying goals and objectives, problems and solutions, timeline, delivery modalities, measures of success, in a specification document.
```

Comment:
"To design effective educational content, begin by conducting thorough research. early your overall preparation." 
This doesn't seem to work?